llvm-bcanalyzer: add page (#6558)
tldr-pages/tldr: pages/common/llvm-bcanalyzer.md

# llvm-bcanalyzer

> LLVM Bitcode (`.bc`) analyzer.
> More information: <https://llvm.org/docs/CommandGuide/llvm-bcanalyzer.html>.

> - Print statistics on a Bitcode file:

Reviewer: ```suggestion
- Print statistics about a Bitcode file:
```

`llvm-bcanalyzer {{path/to/file.bc}}`

> - Print an SGML representation of and statistics on a Bitcode file:

Reviewer: ```suggestion
- Print an SGML representation and statistics about a Bitcode file:
```
